README.md

```diff
@@ -2,4 +2,18 @@
 
```

Comment:
Add:

"This repository contains the documentation and usage instructions for the mybinder.org service"